### ISSUE TITLE
Update to 7.0.4

### ISSUE THREAD

**Issue by NathanSweet** [PolarGoose/Ragel-for-Windows#1]
Any chance we can get a Windows build for the latest, 7.0.4?

**Comment by PolarGoose** [comment]
Hello @NathanSweet,

Thank you for your interest.
This is something I thought about myself.
Unfortunately, I think it will take a lot of effort. The build process of Ragel 7.x is much more complex. Some of the blockers that I see:
* Ragel depends on [Colm](https://github.com/adrian-thurston/colm/tree/master), which is also not available for Windows.
* It seems that the Colm has a runtime dependency on GCC.
* The build process is much more complicated.

Therefore, I don't think I can adapt the Ragel 7's build system to produce a Windows binary as I did for Ragel 6.
I tried to use `mingw-w64` on Linux to attempt to produce a Windows binary without making any changes in Ragel's build scripts. Unfortunately, I couldn't compile the Colm project this way.
